content/apps/govcloud.md

```diff
@@ -2,13 +2,13 @@
 menu:
   main:
     parent: apps
-title: What Is New in Govcloud?
+title: What's New in Cloud.gov?
 ---
 
-The new environment is a bit different and full of shiny new things here is some and how use them:
+The new environment is a bit different and full of shiny new things.  Here are some and how to use them:
 
-The new API endpoint (for now) is api.fr.cloud.gov. To use it: `cf login -a api.fr.cloud.gov --sso`. Your GSA/EPA credentials should still work there, no other credentials work at this time.
+The new API endpoint (for now) is `api.cloud.gov`. To use it: `cf login -a api.cloud.gov --sso` for GSA/EPA accounts and `cf login -a api.cloud.gov` for non-GSA/EPA accounts. See [Setting Up Your Account]({{< relref "../getting-started/accounts.md" >}}) for more information.
```

Comment:
`api.fr.cloud.gov` is correct—we have the two environments running simultaneously, and thus need a different hostname for the API.